3d Space Shooter/Assets/MainMenu.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MainMenu : MonoBehaviour {
```

Comment:
So Looks like you have two MainMenu.cs files in different folders, was this intentional?